data/container-workload.service

```diff
@@ -3,6 +3,7 @@ Description=Container Workload
 After=opt-rootfs-resolv.service
 After=opt-rootfs-proc.mount
 After=opt-rootfs-sys.mount
+After=hyperstart.service
```

Comment:
good catch!,  thinking about this we will want that hyperstart start after shared rootfs mount? or the daemon will mount the shared rootfs (and will remove opt-rootfs.mount )?